### DISCUSSION TITLE
Super Cluster and Operators

### DISCUSSION THREAD

**Discussion by jochen123456** [nats-io/nats-server#5348] [Q&A]
Hi,

is it possible to generate a super cluster out of two cluster - cluster A with Operator A and  cluster B with Operator B?

If yes how will import and exports defined in such construct. For example, pub in cluster A on test and sub in Cluster B on test.

Cheers Jochen

**Comment by derekcollison** [comment]
A NATS system does support multiple operators. It does not support multiple JetStream domains per se (sans the extended NATS systems with leafnodes).

However, if you are asking to export and import from accounts that are signed by different operators, that should be able to work but could involve a few steps.

**Reply by maurobalbi** [reply]
Any pointers as to what those steps would be?
